numpy/core/tests/test_numerictypes.py

```diff
@@ -88,9 +88,7 @@ def normalize_descr(descr):
                 nitem = (item[0], dtype)
             out.append(nitem)
         elif isinstance(item[1], list):
-            l = []
-            for j in normalize_descr(item[1]):
-                l.append(j)
+            l = normalize_descr(item[1])
```

Comment:
While you're changing this line, may as well replace `item[1]` with `dtype` from line 77